kernel/src/callback.rs

```diff
@@ -16,7 +16,7 @@ impl AppId {
     }
 }
 
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
Here I'm a bit weary of `Debug` exposing `appdata` and `fn_ptr` exposed through Debug, but then again might be particularly those fields that are useful